code/modules/keybindings/robot.dm

```diff
@@ -1,3 +1,20 @@
+/datum/keybinding/silicon
+	category = CATEGORY_ROBOT
+
+/datum/keybinding/silicon/toggle_lights
+	hotkey_keys = list("V")
+	name = "toggle_lights"
+	full_name = "Toggle Lights"
+	description = "Toggle camera lights"
+
+/datum/keybinding/silicon/toggle_lights/down(client/user)
+	if(isrobot(user.mob))
+		var/mob/living/silicon/robot/S = user.mob
+		S.toggle_lights()
```

Comment:
This probably won't be a problem, but I would rather you have this as an if and then if else rather than two ifs. This shouldn't happen, but you never know if both statements could run.